src/main/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExt.kt

```diff
@@ -76,3 +78,14 @@ data class EagerResponse(
  * @author sergey
  */
 data class Header(val name: String, val value: String)
+
+internal val stringMapper = ObjectMapper()
+fun String?.asJson(): JsonNode = if (isNullOrBlank()) stringMapper.readTree("{}") else stringMapper.readTree(this)
```

Comment:
This ext function looks `private` and not aligned with `Response` (b/c `Response.asJson` will be used)
Probably it would be better to combine this method with `Response.asJson`

Reply:
I saw usage in tests but not sure if it's useful for end users